source/agora/utils/Log.d

```diff
@@ -115,3 +115,43 @@ public class AgoraLayout : Appender.Layout
         }
     }
 }
+
+/***************************************************************************
+
+    Logging function which is only called from C++ code
+
+    It's for C++ code to use agora's logger instead of C++ stdout 
+
+    Params:
+        logger = the logger name
+        level = the logging level
+        msg = the log message
+
+***************************************************************************/
+private extern (C++) void DLogging (const char* logger, int level, const char* msg)
+{
+    import std.string;
+
+    auto log = Log.lookup(fromStringz(logger));
+    if (log !is null)
+    {
+        log.format(cast(Level) level, fromStringz(msg));
+    }
+}
+
+version(unittest)
```

Comment:
I think the `logTestFromD` and the associated unittest are unnecessary here. They're not actually testing anything other than doing a function call, which the SCP code already does. For example the `[SCP] "Hello, world 1 99.9"` string is not actually tested anywhere in this unittest.

Since the logging code is fairly trivial, I think it's ok not to add tests for it. We will notice if logging fails.

CC @Geod24